web/static/style/design.css

```diff
@@ -179,6 +179,9 @@ a.row {
 .items-start {
   align-items: flex-start;
 }
+.no-shrink {
```

Comment:
is this the name tailwind uses? in general, if something is equivalent to tailwind utilities it makes sense to use their naming.

Reply:
oops, i forgot about following tailwind names. fixed!